src/rgw/rgw_basic_types.h

```diff
@@ -649,5 +241,54 @@ inline std::ostream& operator<<(std::ostream& out, const rgw_user &u) {
   return out << s;
 }
 
+#include "rgw_obj_manifest.h"
+
+struct RGWUploadPartInfo {
+  uint32_t num;
+  uint64_t size;
+  uint64_t accounted_size{0};
+  std::string etag;
+  ceph::real_time modified;
+  RGWObjManifest manifest;
+  RGWCompressionInfo cs_info;
+  std::vector<std::string> saved_prefixes; // Yixin's preferred member name
```

Comment:
As much as I like to be famous, this comment may not provide any value here. :)

Reply:
do you intend to leave this mpu stuff in the PR?